x-pack/plugins/security_solution/public/detection_engine/rule_response_actions/osquery/osquery_investigation_guide_panel.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, EuiFlexGroup, EuiFlexItem, EuiPanel, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React, { useCallback, useState } from 'react';
+
+interface OsqueryInvestigationGuidePanelProps {
+  onClick: () => void;
+}
+
+export const OsqueryInvestigationGuidePanel: React.FC<OsqueryInvestigationGuidePanelProps> =
+  React.memo(({ onClick }) => {
+    const [hideInvestigationGuideSuggestion, setHideInvestigationGuideSuggestion] = useState(false);
+
+    const handleClick = useCallback(() => {
+      onClick();
+      setHideInvestigationGuideSuggestion(true);
+    }, [onClick]);
+
+    if (hideInvestigationGuideSuggestion) {
+      return null;
+    }
+    return (
+      <EuiPanel color={'primary'} paddingSize={'xs'} css={{ marginBottom: '16px' }}>
+        <EuiFlexGroup direction={'row'} alignItems={'center'} css={{ padding: `0 24px` }}>
+          <EuiFlexItem grow={true}>
+            <EuiText size="s">
+              <FormattedMessage
+                id="xpack.securitySolution.responseActionsList.investigationGuideSuggestion"
+                defaultMessage="It seems that you have suggested queries in investigation guide, would you like to add them as response actions?"
```

Comment:
@gchaps could you help us with the message here? 🙏

Reply:
Fantastic, thank you!